lib/attributor/attribute.rb

```diff
@@ -296,6 +312,8 @@ def check_options!
 
     # TODO: override in type subclass
     def check_option!(name, definition)
+      return check_custom_option(name, definition) if self.class.custom_options.include? name
```

Comment:
IF the custom option exists in the type already...should this override it? I feel it might be the other way around? i.e., if a type supports a given option, it should override any general custom option of the attribute class no?
i.e., move the line "after" we check the native options first? 
what do you think?

Reply:
You mean if the existing custom option has the same name as one that already exists?